index.html

```diff
@@ -22,7 +22,20 @@ <h1>Interactive Photo Gallery</h1>
 
   <section id="gallery" class="gallery">
     <!-- Photos will be dynamically added here -->
-
+     <img src="https://plus.unsplash.com/premium_photo-1673292293042-cafd9c8a3ab3?q=80&w=987&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D">
```

Comment:
This homework was intended to not edit the HTML, but to use the `photos` array to create `<img/>` in javascript using `document.createElement('img')`. You should edit this submission to do that.